generic3g/FieldDictionary.F90

```diff
@@ -229,11 +223,24 @@ function get_standard_name(this, alias, rc) result(standard_name)
       _RETURN(_SUCCESS)
    end function get_standard_name
 
+   function get_regrid_method(this, standard_name, rc) result(regrid_method)
+      class(FieldDictionary), target, intent(in) :: this
+      character(*), intent(in) :: standard_name
+      integer, optional, intent(out) :: rc
+      type(ESMF_RegridMethod_Flag), allocatable :: regrid_method ! result
```

Comment:
Please move this ahead of the dummy arguments.  Style is debatable - but consistency ...

Reply:
This, I'm not sure of. The arguments and result are specified left to right, as they appear in the API. I'm going to leave it this way for now.

Reply:
Consider functions like the following:
```
integer function  f(x, y, ...)
end function f
```
Here the return type is declared prior to the dummy arguments.

But really my motivation is simply that the return type of a procedure is often of more interest than its arguments.   

In any event, at this point, I think consistency across procedures is the highest priority, and for almost all of MAPL3, I have declared the return value first.

Reply:
Ok (grudgingly). Commit `f491ba3c` has the changes.